src/Components/UpcomingTable/UpcomingTable.tsx

```diff
@@ -328,17 +328,20 @@ export const UpcomingTable: React.FunctionComponent<UpcomingTableProps> = ({
             {filteredData.length > 0 && (
               <Th>
                 <span className="pf-v5-c-table__td pf-v5-c-table__toggle">
-                  <Button icon={<div className="pf-v5-c-table__toggle-icon">
-                      <AngleDownIcon />
-                    </div>}
+                  <Button
+                    icon={
+                      <div className="pf-v5-c-table__toggle-icon">
```

Comment:
I have updates of css for the upcoming already [there](https://github.com/pr-watson/digital-roadmap-frontend/pull/8). Do you mind I would create new PR against this branch (I would remove the updates of lifecycle page from it since it's part of this PR)? So we can merge both at once.